mlflow/metrics/__init__.py

```diff
@@ -263,6 +264,32 @@ def rougeLsum() -> EvaluationMetric:
     )
 
 
+precision_at_k = make_metric(
```

Comment:
Can we convert this into a function similar to all the other metrics for consistency?